man/fabric.7.md

```diff
@@ -300,23 +300,13 @@ portability across providers.
   on write restrictions.
 
 ## CUDA deadlock
-In some cases, calls to `cudaMemcpy` within libfabric may result in a
-deadlock. This typically occurs when a CUDA kernel blocks until a
-`cudaMemcpy` on the host completes.  To avoid this deadlock,
-`cudaMemcpy` may be disabled by setting
-`FI_HMEM_CUDA_ENABLE_XFER=0`. If this environment variable is set and
```

Comment:
This was the only reference I could find to `FI_HMEM_CUDA_ENABLE_XFER` in the man pages, but it doesn't feel appropriate to add a deprecation notice in this section.